.github/workflows/framework-kind.yml

```diff
@@ -1,90 +1,12 @@
-name: Policy Framework KinD tests
+name: Framework
 
 on:
   pull_request:
     branches:
       - main
-      - release-2.[5-9]
-
-defaults:
-  run:
-    shell: bash
+      - release-2.[7-9]
 
 jobs:
-  kind-tests:
-    runs-on: ubuntu-latest
-    env:
-      REGISTRY: localhost:5000
-    strategy:
-      matrix:
-        # Run tests on oldest and newest supported OCP Kubernetes
-        # - OCP 4.5 runs Kubernetes v1.18
-        # KinD tags: https://hub.docker.com/r/kindest/node/tags
-        kind:
-          - 'v1.18.15'
-          - 'latest'
-        deployOnHub:
-          - 'true'
-          - 'false'
-
-    name: Policy Framework KinD tests
-    steps:
-    - name: Checkout Cert Policy Controller
-      uses: actions/checkout@v2
-      with:
-        path: cert-policy-controller
-        fetch-depth: 0 # Fetch all history for all tags and branches
-
-    - name: Checkout Policy Framework
-      uses: actions/checkout@v2
-      with:
-        path: governance-policy-framework
-        repository: stolostron/governance-policy-framework
-        ref: ${{ github.event.pull_request.base.ref }}
-
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version-file: governance-policy-framework/go.mod
-
-    - name: Policy Framework cluster bootstrap
-      working-directory: governance-policy-framework
-      env:
-        deployOnHub: ${{ matrix.deployOnHub }}
-      run: |
-        make e2e-dependencies
-        make kind-bootstrap-cluster
-
-        echo "* Listing all pods on cluster..."
-        kubectl get pods -A
-
-        # save kubeconfig paths for use in other steps
-        echo "MANAGED_KUBECONFIG=$(pwd)/kubeconfig_managed" >> $GITHUB_ENV
-        echo "HUB_KUBECONFIG=$(pwd)/kubeconfig_hub" >> $GITHUB_ENV
-
-    - name: Patch image
-      working-directory: cert-policy-controller
-      env:
-        deployOnHub: ${{ matrix.deployOnHub }}
-        WATCH_NAMESPACE: managed
-      run: |
-        export KUBECONFIG=${MANAGED_KUBECONFIG}
-        if [[ "${deployOnHub}" == "true" ]]; then
-          export KIND_NAME=hub
-        else
-          export KIND_NAME=managed
-        fi
-
-        make build-images
-        make kind-deploy-controller-dev
-
-    - name: Policy Framework e2e tests
-      working-directory: governance-policy-framework
-      run: |
-        make e2e-test
-
-    - name: Debug
-      if: ${{ failure() }}
-      working-directory: governance-policy-framework
-      run: |
-        make e2e-debug
+  fw-kind-tests:
+    name: KinD
+    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main
```

Comment:
This is fine for now since `main` and `release-2.7` are the same, but can this be templated to match the target branch? I know there are limited places the templates work, so that may not be possible...

Reply:
Oh, that would be a slick way to do it, let me try

Reply:
And if so, I can have github kick off another ~30 tests 😆

Reply:
This page doesn't make me hopeful: https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

We might have to update it manually each release.

Reply:
yeah 😓 https://github.com/stolostron/cert-policy-controller/actions/runs/2929405465

Reply:
😢 Oh well, it was worth a shot!